client/v2/tx/tx.go

```diff
@@ -1,91 +1,108 @@
 package tx
 
 import (
-	"bufio"
 	"context"
 	"errors"
 	"fmt"
-	"os"
-
-	"github.com/cosmos/gogoproto/proto"
-	"github.com/spf13/pflag"
 
 	apitxsigning "cosmossdk.io/api/cosmos/tx/signing/v1beta1"
+	clientcontext "cosmossdk.io/client/v2/autocli/context"
 	"cosmossdk.io/client/v2/broadcast"
 	"cosmossdk.io/client/v2/broadcast/comet"
 	"cosmossdk.io/client/v2/internal/account"
+	"cosmossdk.io/client/v2/internal/flags"
 	"cosmossdk.io/core/transaction"
+	"github.com/cosmos/gogoproto/grpc"
+	"github.com/cosmos/gogoproto/proto"
+	"github.com/spf13/pflag"
 
-	"github.com/cosmos/cosmos-sdk/client"
-	"github.com/cosmos/cosmos-sdk/client/input"
-	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/codec"
 )
 
 // GenerateOrBroadcastTxCLIWithBroadcaster will either generate and print an unsigned transaction
```

Comment:
I guess it should be GenerateAndBroadcast... now, as we've killed the signing confirmation.
If a the caller wants to ask the user to ask he should: call genetate, ask for confirmation, call broadcast.